mno_analysis_tools/compute_msg_difference_btwn_two_firebase_time_periods.py

```diff
@@ -102,9 +102,9 @@ def date_time_range(start, end, delta):
             "Operator": operator,
             "MessageDirection": msg_direction,
             "PeriodStart": messages_per_two_firebase_time_period[index]["PeriodStart"],
-            "PeriodBetween": messages_per_two_firebase_time_period[index]["PeriodEnd"],
+            "PeriodBetween": messages_per_two_firebase_time_period[index]["PeriodBetween"],
             "PeriodEnd": messages_per_two_firebase_time_period[next_index]["PeriodEnd"],
-            "MessageDifference": messages_per_two_firebase_time_period[next_index]["NumberOfMessages"] - messages_per_two_firebase_time_period[index]["NumberOfMessages"]
+            "MessageDifference": str(abs(messages_per_two_firebase_time_period[next_index]["NumberOfMessages"] - messages_per_two_firebase_time_period[index]["NumberOfMessages"]))
```

Comment:
Are you sure you want the abs value? I think it might be useful to tell the difference between when the number of messages is increasing vs decreasing

Reply:
Why is this number being converted to a string and not the others?

Reply:
> Are you sure you want the abs value? I think it might be useful to tell the difference between when the number of messages is increasing vs decreasing

Ok. I think it will help.